### DISCUSSION TITLE
Clone Problem

### DISCUSSION THREAD

**Discussion by billybena** [LianqiangGuan/Tafe230211#12] [General]
Hi Team,

I am still not able to clone the repository correctly. After I cloned the repo to my local machine, I'd tried to switch/checkout to the development branch, but the list only shows master and remote branches. There is no development branch. I got my calculator working, it is just the clone part I am having problems with.

**Comment by LianqiangGuan** [comment]
I just sent you an email to solve this.  Hope that could help you.

**Reply by billybena** [reply]
Thanks Lian. It is working now.

**Comment by maryamabasova** [comment]
Hi,

It sounds like you're facing an issue with cloning the repository and switching to the development branch. Here are some steps you can take to troubleshoot the problem:

1. Check Remote Branches: Ensure that the development branch exists in the remote repository.
2. Fetch Branches: Make sure you have fetched the latest changes from the remote repository. 
3. Checkout Remote Branch
4. Check Permissions: Ensure that you have the necessary permissions to access the development branch. Sometimes, access restrictions can prevent you from seeing or checking out certain branches.
5. Repository URL: Double-check that you've cloned the correct repository and are using the correct repository URL.

If you continue to face issues, please provide more specific details about your setup and any error messages you're encountering so that we can assist you further.

**Reply by billybena** [reply]
Thanks Maryam for your reply. I got the clone working now.
